packages/icons/icons.json

```diff
@@ -4352,5 +4352,12 @@
         "tags": "sync, storage, data, source, dataset, data source, index, rows, columns, agenda, list, spreadsheet, table, sheet, entry",
         "group": "table",
         "codepoint": 62318
+    },
+    {
+        "displayName": "Database search",
+        "iconName": "database-search",
```

Comment:
noting this differs from existing database icons that have some decoration
![Screenshot 2024-07-24 at 9 41 29 AM](https://github.com/user-attachments/assets/c655d42e-39c1-490e-a3bc-b2d4a9e37a80)
![Screenshot 2024-07-24 at 9 41 24 AM](https://github.com/user-attachments/assets/2e66a898-50e8-4daa-aa1f-e3b1397ce3b3)

but the clean `database` icon is named "database" so no clear preference here. my vote is to align this with the base icon and considers the others slightly off-named